tests/e2e/cluster_downgrade_test.go

```diff
@@ -135,7 +162,7 @@ func testDowngradeUpgrade(t *testing.T, clusterSize int, triggerSnapshot bool, t
 		t.Logf("Cancelling downgrade before enabling")
 		e2e.DowngradeCancel(t, epc)
```

Comment:
When I was running e2e tests locally, the `DowngradeCancel` call times out (`Error: etcdserver: request timed out`) quite often, esp. with more nodes downgraded in the cluster.